extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson-common/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/common/runtime/ClientObjectMapper.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.resteasy.reactive.jackson.common.runtime;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.enterprise.util.AnnotationLiteral;
+import javax.inject.Qualifier;
+
+/**
+ * Qualifier Annotation that specializes production, customization, and injection of `client` ObjectMapper
```

Comment:
I think we should also mention what the purpose of this `client` ObjectMapper is - i.e. that it is only used for the reactive rest client

Reply:
done